api/api_ethereum.go

```diff
@@ -479,7 +479,7 @@ func (api *EthereumAPI) GetBlockByNumber(ctx context.Context, number rpc.BlockNu
 func (api *EthereumAPI) GetBlockByHash(ctx context.Context, hash common.Hash, fullTx bool) (map[string]interface{}, error) {
 	klaytnBlock, err := api.publicBlockChainAPI.b.BlockByHash(ctx, hash)
 	if err != nil {
-		return nil, err
+		return nil, nil
```

Comment:
How about a comment that this is intentional?

Reply:
@ian0371 
Sounds great :) 
I'll add some comment for it. Thanks.

Reply:
I added comments in [this commit](https://github.com/klaytn/klaytn/pull/1228/commits/e1452d4963ebdb777e170ef0a914dedb1429ec0a).
Please take a look.

Reply:
LGTM thx :)